src/amo/components/CategoriesHeader/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+.CategoriesHeader .Card-contents {
+  display: flex;
+}
+
+.CategoriesHeader-loadingText {
+  @include margin-end(5px);
+}
+
+.CategoriesHeader-list {
+  display: block;
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
```

Comment:
I see a couple instances of `padding: 0` and `margin: 0` in this patch. Are all those necessary? It didn't seem like they were necessary for overrides.